content/tutorials/how-to-test-contracts/10.index.md

```diff
@@ -100,34 +128,71 @@ it's necessary to use the `hardhat-chai-matchers` plugin.
 
 In the root directory of your project, execute this command:
 
+<span id="install-chai-ethers" data-name="runCommand" data-command-folder="tests-output/hardhat-project" data-pre-command="<COMMAND>"></span>
+
 ::code-group
 
 ```bash [npm]
-npm i -D @nomicfoundation/hardhat-chai-matchers chai@4.3.6 @nomiclabs/hardhat-ethers
+npm i -D @nomicfoundation/hardhat-chai-matchers chai@4.3.6 @matterlabs/hardhat-zksync
 ```
 
 ```bash [yarn]
-yarn add -D  @nomicfoundation/hardhat-chai-matchers chai@4.3.6 @nomiclabs/hardhat-ethers
+yarn add -D  @nomicfoundation/hardhat-chai-matchers chai@4.3.6 @matterlabs/hardhat-zksync
 ```
 
 ::
 
-After installing it, add the plugin at the top of your `hardhat.config.ts` file:
+After installing it, add the plugins at the top of your `hardhat.config.ts` file:
+
+<span id="import-chai-matchers" data-name="modifyFile" data-filepath="tests-output/hardhat-project/hardhat.config.ts"
+data-at-line="4"></span>
 
 ```ts [hardhat.config.ts]
-import "@nomicfoundation/hardhat-chai-matchers"
+import "@matterlabs/hardhat-zksync";
+import "@nomicfoundation/hardhat-chai-matchers";
+```
+
+With the previous steps completed, your `hardhat.config.ts` file should now be properly configured to include settings for local testing.
+
+<span id="compare-config" data-name="compareToFile" data-filepath="tests-output/hardhat-project/hardhat.config.ts"></span>
+
+```ts [hardhat.config.ts]
+import { HardhatUserConfig } from "hardhat/config";
+import "@nomicfoundation/hardhat-toolbox";
+import "@matterlabs/hardhat-zksync-node";
+import "@matterlabs/hardhat-zksync";
```

Comment:
```suggestion
import "@matterlabs/hardhat-zksync";
```

hardhat-zksync already includes the node so you don't need to import it. Actually you can probably skip one of the steps above as well